src/rpc/util.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_RPC_UTIL_H
+#define BITCOIN_RPC_UTIL_H
+
+#include "keystore.h"
```

Comment:
Could drop this include and just forward-declare CKeyStore. Might be good to add includes for \<string\> and \<vector\>.

Reply:
I tried forward declaring CKeytStore and it didn't work.

Reply:
Following change should work:

```diff
diff --git a/src/rpc/util.h b/src/rpc/util.h
index f933b3da92..18f38e8cd3 100644
--- a/src/rpc/util.h
+++ b/src/rpc/util.h
@@ -5,8 +5,7 @@
 #ifndef BITCOIN_RPC_UTIL_H
 #define BITCOIN_RPC_UTIL_H
 
-#include "keystore.h"
-
+class CKeyStore;
 class CPubKey;
 class CScript;
 
diff --git a/src/rpc/util.cpp b/src/rpc/util.cpp
index 2ebec303cb..a288fe75f5 100644
--- a/src/rpc/util.cpp
+++ b/src/rpc/util.cpp
@@ -3,6 +3,7 @@
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 #include "base58.h"
+#include "keystore.h"
 #include "pubkey.h"
 #include "tinyformat.h"
 #include "rpc/protocol.h"
```

Reply:
Done